### PR TITLE
Add react resources

### PR DESCRIPTION
**Pull request by hamzaazman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -456,6 +456,8 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [React Patterns](https://reactpatterns.com/)
 - [Useful Packages you should know in React](https://medium.com/better-programming/14-useful-packages-every-react-developer-should-know-55b47a325d3)
 - [Learn React Test Driven Development](https://learntdd.in/react/)
+- [React.js Cheatsheet](https://devhints.io/react)
```
Comment (shahednasser) [inline README.md:459]: This resource is already added so please remove it

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi @hamzaazman thank you for your contribution, I just have a comment below on an existing resource

File: README.md (hunk 1 above)
Comment (hamzaazman) [reply README.md:459]: Oh, sorry. I'm fixing it.

**Comment by shahednasser** [comment]
Thank you!

**Pull request merged**